src/BenchmarkDotNet/Configs/ConfigExtensions.cs

```diff
@@ -132,5 +134,4 @@ private static ManualConfig With(this IConfig config, Action<ManualConfig> addAc
             addAction(manualConfig);
             return manualConfig;
         }
-    }
 }
```

Comment:
this should fix the build

```suggestion
    }
}
```